### PR TITLE
GH Actions service change from mysql to mariadb

### PR DESCRIPTION
**Pull request by davidbeig**
Change gh actions service from mysql to mariadb

### CODE REVIEW

**Review by microstudi — COMMENTED** [review]
I think configuration for mariadb is a little bit different, something like this:

image

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -37,10 +37,11 @@ jobs:
           --health-retries=3
 
     steps:
-      - name: Verify MySQL connection from host
+      - name: Verify MariaDB connection from host
```
Comment (microstudi) [inline .github/workflows/test.yml:40]: an this step is probably all innecessary...

**Comment by davidbeig** [comment]
Okay! I'll change it later. I was trying now to make it work on locally using [@nektos/act](https://github.com/nektos/act)

**Comment by microstudi** [comment]
@davidbeig nice repo! didn't know it!

**Comment by davidbeig** [comment]
Should update this PR with #316 or just close it.

What do you think? @microstudi @carlos-mg89

**Comment by carlos-mg89** [comment]
> Should update this PR with #316 or just close it.
> 
> What do you think? @microstudi @carlos-mg89

I think that we should have either MySQL or MariaDB in all environments.

**Comment by microstudi** [comment]
> I think that we should have either MySQL or MariaDB in all environments.

All MariaDB, we should change any mention to mysql in the docs as well

**Pull request closed without merging**